src/views/annual-report/annual-report.jsx

```diff
@@ -0,0 +1,1279 @@
+                    )}
+                    onClick={this.handleSubnavItemClick(SECTIONS.mission)}
+                >
+                    <FormattedMessage id="annualReport.subnavMission" />
+                </a>
+                <a
+                    className={classNames(
+                        {selectedItem: this.state.currentlyVisible === SECTIONS.milestones}
+                    )}
+                    onClick={this.handleSubnavItemClick(SECTIONS.milestones)}
+                >
+                    <FormattedMessage id="annualReport.subnavMilestones" />
+                </a>
+                <a
+                    className={classNames(
+                        {selectedItem: this.state.currentlyVisible === SECTIONS.reach}
+                    )}
+                    onClick={this.handleSubnavItemClick(SECTIONS.reach)}
+                >
+                    <FormattedMessage id="annualReport.subnavReach" />
+                </a>
+                <a
+                    className={classNames(
+                        {selectedItem: this.state.currentlyVisible === SECTIONS.initiatives}
+                    )}
+                    onClick={this.handleSubnavItemClick(SECTIONS.initiatives)}
+                >
+                    <FormattedMessage id="annualReport.subnavInitiatives" />
+                </a>
+                <a
+                    className={classNames(
+                        {selectedItem: this.state.currentlyVisible === SECTIONS.financials}
+                    )}
+                    onClick={this.handleSubnavItemClick(SECTIONS.financials)}
+                >
+                    <FormattedMessage id="annualReport.subnavFinancials" />
+                </a>
+                <a
+                    className={classNames(
+                        {selectedItem: this.state.currentlyVisible === SECTIONS.supporters}
+                    )}
+                    onClick={this.handleSubnavItemClick(SECTIONS.supporters)}
+                >
+                    <FormattedMessage id="annualReport.subnavSupporters" />
+                </a>
+                <a
+                    className={classNames(
+                        {selectedItem: this.state.currentlyVisible === SECTIONS.leadership}
+                    )}
+                    onClick={this.handleSubnavItemClick(SECTIONS.leadership)}
+                >
+                    <FormattedMessage id="annualReport.subnavLeadership" />
+                </a>
+                <a
+                    className={classNames(
+                        {selectedItem: this.state.currentlyVisible === SECTIONS.donate}
+                    )}
+                    onClick={this.handleSubnavItemClick(SECTIONS.donate)}
+                >
+                    <FormattedMessage id="annualReport.subnavDonate" />
+                </a>
+            </FlexRow>);
+
+        return (
+            <div>
+                <div
```

Comment:
We should probably pull the subnav into a separate component, after the launch of the annual report.

Either way, the way we are handling responsiveness here might be a little confusing. For small screens, we render both the section indicator (in the "top bar") and the drop down of list of sections (in the "bottom bar.") For small screens, we render only the list of sections, in the top bar (with different styling)